rpcs3/Emu/Cell/Modules/cellMic.h

```diff
@@ -360,7 +362,7 @@ class microphone_device
 	static constexpr usz inbuf_size = 400000; // Default value unknown
 
 	simple_ringbuf<inbuf_size> rbuf_raw;
-	simple_ringbuf<inbuf_size> rbuf_dsp;
+	simple_ringbuf<inbuf_size * 2> rbuf_dsp;
```

Comment:
I think we should move
`float_buf_size`
from convert_16_bit_pcm_to_float to below
`static constexpr usz inbuf_size = 400000; // Default value unknown`
And change it to calculate the multiplier with sizeof expressions
`static constexpr usz float_buf_size = inbuf_size * (sizeof(f32) / sizeof(s16));`

```suggestion
	simple_ringbuf<float_buf_size> rbuf_dsp;
```